specs/common/knative-serving-conformance-testing-workflow.md

```diff
@@ -0,0 +1,34 @@
+# Knative Conformance Workflow
+
+This document covers the workflow steps to gain knative conformance
+
+## Conformance Steps
+1. Run Manual Tests
+2. Open Github PR in knative/specs repo 
+  - Use PR template below
+  - place in the following directory structure
+    - conformance/tests/results/$spec_version/${date}-${product} directory with a metadata file and test results
+    - ex; .../v1.0/2021-12-18-ibm-code-engine/...
+  - Attach Logs from tests
+3. KTC reviews 
+  - Template
+  - logs
+4. KTC provide approval / exception / or non-approval
+5. Post to conformance matrix - consumable by the general public as a table of version, product, vendor and outcomes.
+  - Should have a public “conformant implementations” showcase on https://knative.dev/offerings
```

Comment:
https://knative.dev/offerings does not exist yet, before merging the PR we should ensure that this page exists at least as a placeholder.

Also, an example of a "showcase" would be cool to have as a blueprint, because I've not really an idea how such a _conformant implementation showcase_ should look like.

Reply:
I am not sure how the community wanted that to look.  My thoughts would be to add those details later.

Reply:
ok, so maybe add this as a comment in parentheses, like in "(tbd, details need to be determined)" ?

Reply:
added the (tbd) to the line, we can adjust this later